### DISCUSSION TITLE
Add Chat Highlight for Raiders like in regular Twitch chat

### DISCUSSION THREAD

**Discussion by torresromain** [Chatterino/chatterino2#5339] [Ideas]
One thing I loved with the default Twitch chat on OBS was knowing which chatters came from a raid but this doesn't exist in Chatterino, I would love to have the option to add it as a yellow highlight for messages.

[This is the option in Twitch Chat settings.](https://github.com/Chatterino/chatterino2/assets/47885402/3773456d-b1a4-4056-92bd-84b8f47e4e4d)

 
[And this is how it looks like in chat with a message.](https://github.com/Chatterino/chatterino2/assets/47885402/fdc65386-0bce-49b9-a396-44a010a033bf)

Would be great to have this in Chatterino, thanks!

**Comment by Felanbird** [comment]
I don't believe Twitch offers the info to us of who is a raider

**Reply by iProdigy** [reply]
correct; this would require unofficial pubsub

**Reply by torresromain** [reply]
Oh okay, that's too bad

**Reply by Nerixyz** [reply]
Once we have EventSub, we could probably use [channel.raid](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelraid).

**Reply by iProdigy** [reply]
`channel.raid` unfortunately doesn't tell us the individual chatter user id's that are joining from the raiding channel

**Reply by Nerixyz** [reply]
Right, I didn't read the issue currently. Maybe the frontend migrates to EventSub some day and this info will be available there.
